packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -152,11 +165,17 @@ export class LitDevTutorial extends LitElement {
   private _manifestTask = new Task(
     this,
     async ([projectLocation]) => {
+      this._manifestLoadError = '';
       const manifestRes = await fetch(
         `${this._samplesRoot}/tutorials/${projectLocation}/tutorial.json`
       );
 
-      return (await manifestRes.json()) as TutorialManifest;
+      if (manifestRes.ok) {
+        return (await manifestRes.json()) as TutorialManifest;
+      } else {
+        this._manifestLoadError = `Could not fetch tutorial manifest. Invalid URL /${projectLocation}/`;
```

Comment:
Doesn't this introduce a similar race condition to before? If request A fails but is slow, and request B succeeds but is fast, then the error will get displayed even though it's no longer relevant.

Reply:
In this case the UI makes navigating to these states unlikely. This will only happen if we forgot to deploy some of these files. Anyway, I'd prefer this failure to have high visibility WDYT?

Reply:
tackled this in #667